src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -446,81 +446,20 @@ export class AttachToDropdown extends SelectBox {
 			this.setOptions([msgLocalHost]);
 		}
 		else {
-			let connections = this.getConnections(model);
-			this.enable();
-			if (showSelectConnection) {
-				this.loadWithSelectConnection(connections);
+			let connections: string[] = model.context && model.context.title ? [model.context.title] : [];
+			if (connections.length === 0) {
```

Comment:
nit: Could just have the [] value in the line above be [msgSelectConnection] instead.

Reply:
ooh i like that. thanks.